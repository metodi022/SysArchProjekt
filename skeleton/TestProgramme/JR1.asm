lui $1, 0x1234
jal end
start:				
ori $1, 0x1111
jr $31
end:
jal start