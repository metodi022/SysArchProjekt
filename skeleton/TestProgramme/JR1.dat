0x3c011234
0x0c100004
0x34211111
0x03e00008
0x0c100002
