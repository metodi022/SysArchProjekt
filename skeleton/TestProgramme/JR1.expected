@1 12341111
@1f 00400014